fdbserver/workloads/TenantManagementWorkload.actor.cpp

```diff
@@ -1663,6 +1710,8 @@ struct TenantManagementWorkload : TestWorkload {
 				wait(getTenantGroup(self));
 			} else if (operation == 7) {
 				wait(listTenantGroups(self));
+			} else if (operation == 8) {
+				wait(deleteTenant(self, true));
```

Comment:
Rather than run this as two separate operations, I'd recommend keeping one `deleteTenant` operation and choosing whether to test the watch aspect randomly within the operation.